recipes/setproctitle/bld.bat

```diff
@@ -0,0 +1,2 @@
+"%PYTHON%" setup.py install
```

Comment:
We need to fix this. https://github.com/conda/conda-build/issues/682

I'll try to get around to it in the near future!